tedana/resources/extern_regress_dicts/Fmodel.json

```diff
@@ -0,0 +1,7 @@
+{
+    "regess_ID": "Fmodel",
+    "info": "Fits all external regressors to a single model using an F statistic",
+    "report": "Unspecified external nuissance regressors were fit to components using a linear model were rejected if they fit.",
```

Comment:
I definitely think this info should be in the decision tree. It refers to decisions that are part of the tree, rather than any metric calculation info.

Reply:
Done.